apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTestRunner.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef AUDIO_WORKLOAD_TEST_RUNNER_H
+#define AUDIO_WORKLOAD_TEST_RUNNER_H
+
+#include <string>
+#include <chrono>
+#include <atomic>
+#include <memory>
+#include "AudioWorkloadTest.h"
+
+/**
+ * @class AudioWorkloadTestRunner
+ * @brief Manages the execution and results of an AudioWorkloadTest.
+ *
+ * This class encapsulates an AudioWorkloadTest instance, controls its lifecycle (start, stop),
+ * monitors its progress, and determines a pass/fail result based on criteria like XRuns.
+ *
+ * Example usage:
+ * start();
+ * while(!stopIfDone()) { getStatus(); sleep(); }
+ * getResultText();
+ *
+ */
+class AudioWorkloadTestRunner {
+public:
+    /**
+     * @brief Default constructor.
+     */
+    AudioWorkloadTestRunner() = default;
+
+    /**
+     * @brief Destructor. Ensures the test is stopped when the runner is destroyed.
+     */
+    ~AudioWorkloadTestRunner();
+
+    /**
+     * @brief Starts the audio workload test with the specified parameters.
+     * @param targetDurationMs The desired duration of the test in milliseconds.
+     * @param bufferSizeInBursts The desired buffer size in terms of multiples of framesPerBurst.
+     * @param numVoices The primary number of synthesizer voices to simulate.
+     * @param highNumVoices An alternative number of voices for alternating workload.
+     * @param highLowPeriodMillis The period in milliseconds to alternate between numVoices and highNumVoices.
+     * @param adpfEnabled Whether to enable Adaptive Performance (ADPF) hints.
+     * @param hearWorkload If true, the synthesized audio will be audible; otherwise, it's processed silently.
+     * @return 0 on success, -1 on failure (e.g., test already running, error opening/starting stream).
+     */
+    int32_t start(
+            int32_t targetDurationMs,
+            int32_t bufferSizeInBursts,
+            int32_t numVoices,
+            int32_t highNumVoices,
+            int32_t highLowPeriodMillis,
+            bool adpfEnabled,
+            bool hearWorkload);
+
+    /**
+     * @brief Stops the test if it has completed its run (i.e., AudioWorkloadTest is no longer running).
+     * This is useful for polling and cleaning up once the underlying test finishes due to duration.
+     * @return True if the test is done (either stopped here or previously), false otherwise.
```

Comment:
This looks wrong. The value is initialized as false. When the stream is not started, it returns false.